### ISSUE TITLE
feat : adding Diabetes Prediction in Diabetes Classification folder

### ISSUE THREAD

**Issue by akashlogics** [SrijanShovit/HealthLearning#161]
### Is your feature request related to a problem? Please describe.

Yes, I would like to find the Diabetes Risk Score Calculator which is a Python script that uses a pre-trained machine learning model to predict diabetes risk. and I would like to add this folder 

I would like to work on the issue can you please assign me and label it GSsoc,24

### Describe the solution you'd like along with reference dataset.

[Diabetes Prediction using Scikit-learn] - A Jupyter notebook that demonstrates diabetes prediction using Scikit-learn

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Taranpreet10451** [comment]
Please assign this to me.

**Comment by adityasingh-0803** [comment]
i can do it can you assign it to me under gssoc label
